pyproject.toml

```diff
@@ -0,0 +1,3 @@
+[build-system]
+requires = ["setuptools", "wheel", "scikit-build", "ninja"]
+build-backend = "setuptools.build_meta"
```

Comment:
Should we require a minimum version of setuptools ?

Reply:
Sure, 42 is a good minimum, as 41 was pretty buggy in its support for PEP 517/518.

Reply:
This also got missed from the update.